src/greyVest/utils.js

```diff
@@ -0,0 +1,8 @@
+import { lensify } from '../utils/react'
+import F from 'futil-js'
+
+let binding = (...lens) => ({
+  isOpen: F.view(...lens),
+  onClose: F.off(...lens),
+})
+export let bindLens = lensify(binding, 'open')
```

Comment:
we should be consistent with material and call this `openify` (or rename `openify` in material)